schema_based_on_diagram.sql

```diff
@@ -0,0 +1,61 @@
+CREATE DATABASE clinicDB;
+
+/* tables from the diagrams */
+
+CREATE TABLE
+    patients(
+        id INT PRIMARY KEY,
+        name VARCHAR(250),
+        date_of_birth DATE
+    );
+
+CREATE TABLE
+    treatment(
+        id INT PRIMARY KEY,
+        name VARCHAR(250),
+    );
+
+CREATE TABLE 
+    medical_histories(
+        id INT PRIMARY KEY,
+        admitted_at DATE,
+        FOREIGN KEY (patient_id) REFERENCES patients(id),
+        status VARCHAR(250),
+    );
+
+CREATE TABLE
+    invoice_items(
+        id INT PRIMARY KEY,
+        unit_price DECIMAL(10,2),
+        quantity INT,
+        FOREIGN KEY (invoice_id) REFERENCES patients(id),
+        FOREIGN KEY (treatment_id) REFERENCES treatment(id),
+    );
+
+CREATE TABLE
+    invoices(
+        id INT PRIMARY KEY,
+        total_amount DECIMAL(10,2),
+        generated_at DATE,
+        payed_at DATE,
+        FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+    );
+
+/* tables that represent the Many to many relationships */
+
+CREATE TABLE
+    medical_histories_treatments(
+        FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+        FOREIGN KEY (treatment_id) REFERENCES treatment(id)
```

Comment:
- [ ] You did great creating the correct JOIN table. I think the indexes for this JOIN table are missing. Let's add them. 🥸️